doc-bridgepoint/notes/9110_Research_bp_profiling_tool/9110_Research_bp_profiling_tool.int.md

```diff
@@ -0,0 +1,207 @@
+---
+
+This work is licensed under the Creative Commons CC0 License
+
+---
+
+# Research best tool for profiling BridgePoint and define profiling procedure 
+### xtUML Project Implementation Note
+
+1. Abstract
+-----------
+This note describes the results from evaluation of commonly used java profilers against the BridgePoint tool.
+
+2. Document References
+----------------------
+<a id="2.1"></a>2.1 [BridgePoint DEI #9110](https://support.onefact.net/issues/9110) Research best tool for profiling BridgePoint and define profiling procedure  
+<a id="2.2"></a>2.2 [BridgePoint Profiler Research](https://docs.google.com/spreadsheets/d/1PILhtDHFWBIWIWv07Ik1mYs_8G_QLoGVgwZoQWTWcaQ/edit#gid=0) Initial research into profiling tools   
+<a id="2.3"></a>2.3 [BridgePoint Profiler](https://docs.google.com/document/d/1PdtBOeROEGpihp4jrgOM8JSUT57Ymsr9E8G7H2qATZ0/edit) Document describing tools further along with environment settings  
+
+3. Background
+-------------
+3.1 Performance issues with BridgePoint
+
+Notable performance issues have shown within the last year.  The tool needs to be profiled in order to determine the cause.  BridgePoint has never had a dedicated tool for profiling.  We need to determine which tool is best to suit the needs of BridgePoint development.
+
```

Comment:
Add a note about the suggestion provided by EclipseSource. What did they suggest and is they provided the "why" put it too?